Makefile

```diff
@@ -68,6 +68,7 @@ info:
 
 .PHONY: install
 install:
+	pip install --upgrade pip
```

Comment:
I'm neither a Python nor a CI expert, so this may be a silly question, but: will this run the risk of pip changing versions and breaking on us?

Reply:
It certainly could run that risk, but in my experience it's more often the case that pip needs to upgrade in order to successfully complete a build. Though to be fair, I typically don't run it in CI either.

I defer to one of the core maintainers here. Maybe @jselig-rigetti .

Reply:
As long as the `pip` installed is compatible with the python version used (which it should be), this should be fine.